modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordSchemaFileProvider.kt

```diff
@@ -0,0 +1,34 @@
+package com.metalbear.mirrord
+import com.intellij.openapi.vfs.VirtualFile
+import com.jetbrains.jsonSchema.extension.JsonSchemaFileProvider
+import com.jetbrains.jsonSchema.extension.SchemaType
+import com.jetbrains.jsonSchema.impl.JsonSchemaVersion
+import com.jetbrains.jsonSchema.remote.JsonFileResolver
+
+class MirrordSchemaFileProvider: JsonSchemaFileProvider {
+
+    override fun isAvailable(vfile: VirtualFile): Boolean {
+        val path = vfile.path
+        return path.contains("mirrord") && path.endsWith(".json")
```

Comment:
Is this the same condition we have in our config file indexer? We should probably keep these in sync

Reply:
```kotlin
        return FileBasedIndex.InputFilter {
            it.isInLocalFileSystem && !it.isDirectory && (
                it.path.endsWith("mirrord.json") ||
                    it.path.endsWith("mirrord.yaml") ||
                    it.path.endsWith("mirrord.toml")
                )
        }
```
This is the condition from the config indexer. Think we should unify

Reply:
good one